### DISCUSSION TITLE
Code Fold Indicators - Bold horizontal lines when collapsed

### DISCUSSION THREAD

**Discussion by advice2020** [geany/geany#3998] [Ideas]
Suggesting to maybe add another value to `folding_horiz_line` to make the horizontal line that shows up when code fold is collapsed to be a bold/thicker line.

Thank You

The suggestion post above can stand on its own as being a feature, but would also be a requirement if the following post is added
https://github.com/geany/geany/discussions/3987

**Comment by elextr** [comment]
You are limited to what Scintilla allows, see [here](https://www.scintilla.org/ScintillaDoc.html#Folding).  If its possible then "somebody" will need to make a PR to add the option.
